IOPool/Streamer/test/BuildFile.xml

```diff
@@ -5,13 +5,18 @@
   </bin>
   <bin   file="ReadStreamerFile.cpp">
     <use   name="IOPool/Streamer"/>
-    <flags   TEST_RUNNER_ARGS=" bin/bash teststreamfile.dat"/>
+    <use   name="FWCore/Catalog"/>
+    <use   name="FWCore/PluginManager"/>
+    <use   name="FWCore/ServiceRegistry"/>
+    <use   name="FWCore/Services"/>
+    <use   name="FWCore/ParameterSetReader"/>
+    <flags   TEST_RUNNER_ARGS="all"/>
```

Comment:
is `all` a special value known by scram?

Reply:
I see that it is an option known by ReadStreamerFile

Reply:
Right, it's just an arg to the test program, scram doesn't know anything about the value.  Alternatively we could make all the default and get rid of the arg, which might be safer--previously the arg was set to a nonsense string and as a result the tests weren't even being run.